src/garrysmod/gamemodes/zombiesurvival/gamemode/howl/sv_command.lua

```diff
@@ -0,0 +1,95 @@
+local Command = {}
+Command.prototype = {}
+Command.__index = Command.prototype
+
+Command.argument_types = {}
```

Comment:
This is some of the bread and butter of the command system. We're registering wrappers, or argument types, around types of values we want to use as arguments in our commands. For example, a `player` argument type takes in a string and either returns a `Player` object or `nil`. This allows for input validation, and later on, generating automatic usage messages.

Argument types are declared in a separate folder and populate themselves into this table.